pages/index.tsx

```diff
@@ -1,7 +1,14 @@
 import { Inter } from "next/font/google";
+import { useEffect } from 'react';
```

Comment:
```suggestion
import { useEffect } from "react";
```
Please try to set up prettier to automatically format the quotes, such that we consistently have double quotes across the application.